### PR TITLE
Fix #6239: genesis omega field dist

### LINKED ISSUE #6239: show genesis field distribution with other omega plots
When genesis is selected as one of the omega workflow sims, the particle, field and parameter plots should all be shown. Currently the field distribution is missing.

### PR DESCRIPTION
**Pull request by gurhar1133**
Field distribution plot added to genesis visualization in omega

### CODE REVIEW

File: sirepo/template/omega.py [hunk 1]
```diff
@@ -285,12 +287,26 @@ def _report_info(sim_count, model_name, report_count):
...
+                    _report_info(idx + 1, "simFieldDistributionAnimation", 1),
+                ]
+            )
         idx += 1
         sim_dir = _sim_dir(run_dir, idx + 1)
 
     return res
 
 
+def _is_genesis(run_dir, index):
```
Comment (mkeilman) [inline sirepo/template/omega.py:302]: Might move inside `_output_info`

File: sirepo/template/omega.py [hunk 2]
```diff
@@ -285,12 +287,26 @@ def _report_info(sim_count, model_name, report_count):
...
+    t, i = _sim_info(
```
Comment (mkeilman) [inline sirepo/template/omega.py:303]: If you don't need `i` you can replace it with an underscore as a hint to the reader that it isn't used.

**Review by mkeilman — COMMENTED** [review]
Minor comments

File: sirepo/template/omega.py (hunk 2 above)
Comment (gurhar1133) [reply sirepo/template/omega.py:303]: Oh oops. I thought it was the radiasoft convention that we didn't do this.

Comment (mkeilman) [reply sirepo/template/omega.py:303]: I guess I don't know about the convention. I do see it in a number of places...

Comment (gurhar1133) [reply sirepo/template/omega.py:303]: I agree with you that `_` should be the convention for an unused unpacked tuple variable

**Pull request merged**